includes/discovery/sensors/current/apc.inc.php

```diff
@@ -239,14 +238,15 @@
                 echo $item['type'].' '.$item['mib'].' UPS';
             }
 
-            if (stristr($current_oid, 'HighPrec')) {
-                $current = ($oids / $item['divisor']);
+            if ($is_highprec === true) {
```

Comment:
I think it is a bit silly to set a bool then if else later.
Just move the code to set $current up to the if/else where is_highprec is set.

Actually, probably the default divisor should be 1 and set to 10 and divide the current value for high precision